windows-nsis/openvpn.nsi

```diff
@@ -475,10 +472,7 @@ Function .onInit
 
 ${IfNot} ${AtLeastWinVista}
 
-	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES IDNO guiEndNo
-	Goto guiEndYes
-
-	guiEndNo:
+	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES IDYES guiEndYes
 	Quit
 
 	guiEndYes:
```

Comment:
While at it please also change this label to something more meaningful like WinXPYes as @mattock had pointed out earlier.